pkg/config/config.go

```diff
@@ -48,11 +48,11 @@ func (c *Config) InitConfig() {
 		c.File = configFile
 		viper.SetConfigType("toml")
 		viper.SetConfigFile(configFile)
-		viper.SetConfigPermissions(os.FileMode(0600))
+		viper.SetConfigPermissions(os.FileMode(0o600))
```

Comment:
why? did os.FileMode update the way the permissions are passed? or is this coming from formatting.

I can see we use the number with the "o" in the middle, https://pkg.go.dev/os#example-FileMode. I'm just curious of why it used to work and if this is a breaking change